.github/workflows/winget-submission.yml

```diff
@@ -0,0 +1,23 @@
+name: Submit package to Windows Package Manager Community Repository 
+
+on:
+  release:
+    types: [published]
+
+env:
+  version: "$(git describe --tags --abbrev=0)"
```

Comment:
For consistency we should stick to `./scripts/version.sh`. Also, instead of having an evaluated statement here (which doesn't get evaluated instantly), you should just have these be variables in the script instead.